clang/test/CodeGenSYCL/kernel-param-acc-array-ih.cpp

```diff
@@ -0,0 +1,52 @@
+// RUN: %clang -I %S/Inputs -fsycl-device-only -Xclang -fsycl-int-header=%t.h %s -c -o %T/kernel.spv
```

Comment:
Clang driver use is not encouraged in CodeGen or Sema tests. You can use something like
```
 %clang_cc1 -I %S/Inputs -fsycl -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -fsycl-int-header=%t.h %s -fsyntax-only
```
instead.